support FreeBSD and OpenBSD

trombik opened the pull request:
ping(8) flags for timeout vary on platforms.

Commit:
support FreeBSD and OpenBSD

Commit:
make RuboCop happy

otahi commented:
@trombik Thank you very much for your pull request!!
I will merge it soon if following question is clear.
-  Is `-c1 -w3` OK on OpenBSD? I confirmed `-c 1 -w 3` is OK on Linux. 
  - Can we delete OpenBSD case?
  - Should we change default option from `-c1 -w3` to `-c 1 -w 3`?

I will fix CI failure on Ruby 1.9.3 after merging this pull request.

Commit:
bugfix: use -t on FreeBSD

from FreeBSD ping(8)

     -t timeout
             Specify a timeout, in seconds, before ping exits regardless of
             how many packets have been received.

from OpenBSD ping(8)

     -w maxwait
             Specifies the maximum number of seconds to wait for responses
             after the last request has been sent.  The default is 10.

on Linux (Ubuntu), '-w' differs from '-W' but in this context, same.

trombik commented:
the default has changed to '-c 1 -w 3'. as for a space between the flags and value, it is a matter of taste.

otahi commented:
@trombik Sorry for late response. 
I think this modification  is reasonable.
